st2client/st2client/config_parser.py

```diff
@@ -138,6 +141,40 @@ def parse(self):
             # Config doesn't exist, return the default values
             return CONFIG_DEFAULT_VALUES
 
+        config_dir_path = os.path.dirname(self.config_file_path)
+
+        # Make sure the directory permissions == 0o770
+        if bool(os.stat(config_dir_path).st_mode & 0o777 ^ 0o770):
+            self.LOG.warn(
+                # TODO: Perfect place for an f-string
+                "The StackStorm configuration directory permissions are "
+                "insecure (too permissive)."
+                "\n\n"
+                "You can fix this by running:"
+                "\n\n"
+                "chmod 770 {config_dir}".format(config_dir=config_dir_path))
+
+        # Make sure the setgid bit is set on the directory
+        if not bool(os.stat(config_dir_path).st_mode & 0o2000):
+            self.LOG.info(
+                # TODO: Perfect place for an f-string
+                "The SGID bit is not set on the StackStorm configuration "
+                "directory."
+                "\n\n"
+                "You can fix this by running:"
+                "\n\n"
+                "chmod g+s {config_dir}".format(config_dir=config_dir_path))
```

Comment:
For the `LOG.info` message level, do you know/can you check where will it show while running `st2` commands?

Reply:
That depends greatly on the configuration file itself, but the `info` level is not output to stdout or stderr by default:

```python
>>> import logging
>>> logger = logging.getLogger("_")
>>> logger.info("Hello World!")
>>> 
```

Reply:
Thanks for verifying that :+1: